app/views/favorites/index.html.erb

```diff
@@ -2,8 +2,8 @@
 	<h1>Favorite Stops</h1>
 	<% unless current_user.favorites.empty? %>
 		<% current_user.favorites.each do |stop| %>
-			<%= render "stops/show_with_issues", stop: stop %>
-		<% end %>
+		  <%= render "stops/show_with_issues", stop: stop %>
```

Comment:
It looks like some indenting got changed here that shouldn't have been. Could you remedy it?